src/node.cc

```diff
@@ -1291,12 +1236,6 @@ void SetupPromises(const FunctionCallbackInfo<Value>& args) {
 }  // anonymous namespace
 
 
-void AddPromiseHook(v8::Isolate* isolate, promise_hook_func fn, void* arg) {
-  Environment* env = Environment::GetCurrent(isolate);
-  env->AddPromiseHook(fn, arg);
-}
```

Comment:
This should stay, it was intentionally requested for the public API so that Node using the single PromiseHook slot doesn’t block others from doing the same.